packages/zoe/test/unitTests/contracts/test-throwInOfferHandler.js

```diff
@@ -85,6 +85,6 @@ test('throw in offerHandler', async t => {
       // https://github.com/endojs/endo/pull/640
       //
       // /"brand" not found: .*/,
-      /.* not found: .*/,
+      /no ordinal for .*/,
```

Comment:
Is there a TODO or issue somewhere for @FUDCo to have his virtual/durable stores give us a better error message when a key is missing?

Reply:
added #5776